lib/pymedphys/_gamma/implementation/shell.py

```diff
@@ -44,7 +44,6 @@ def gamma_shell(
     skip_once_passed=False,
     random_subset=None,
     ram_available=DEFAULT_RAM,
-    quiet=False,
```

Comment:
Can we deprecate this for now and not remove it just yet. We need a deprecation message that flags that it will be removed in a future version, and that informs the user how to replicate the old functionality with the new logging approach.

Reply:
Sure! So just leave it as a function arg but with no functionality? Or leave in the functionality (i.e. wrap the logs in `if not quiet`)?